src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -687,6 +687,34 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorModel'
+  /api/repository/v1/snapshots/{id}/preview:
+    get:
```

Comment:
I would make this a POST,  not a get.  It's generally not good practice to have bodies in GET requests (and it's common to have POST for data fetching endpoints...e.g. ES does this)

Reply:
This API doesn't have a body (at least not one I can see). The only inputs to the API are UUID, table name, and count.

Reply:
Oh right, sorry, I see that